services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/MetersSpec.groovy

```diff
@@ -50,8 +52,6 @@ class MetersSpec extends BaseSpecification {
     }
 
     def "Default meters should be re-calculated to kbps on Centec switches"() {
-        requireProfiles("hardware")
```

Comment:
this was an intentional duplication, I asked about it in Nikolay's PR. When L34 gets removed, this should remain hardware-only test.